<?xml version="1.0"?>
<launch>
  <node name="stm32_serial" pkg="amr_stm32" type="send_stm32.py" respawn="false" output="screen" />
  <param name="serial_port"         type="string" value="/dev/AMR_STM32"/>
  <param name="serial_baudrate"     type="int"    value="115200"/>
</launch>
